### PR TITLE
Create 733 - Potluck egres, scraping, Safari in EU.md

### PR DESCRIPTION
**Pull request by randyrektor**

### CODE REVIEW

**Comment by wesbos** [comment]
This is good, the markdown script thinks the links are timestamps.

File: shows/733 - Potluck egres, scraping, Safari in EU.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+number: 733
+title: Egress, scraping, Safari EU changes, is SetInterval worth it.mp4
```
Comment (wesbos) [inline shows/733 - Potluck egres, scraping, Safari in EU.md:3]: @randyrektor It says ".mp4" on the end

Comment (randyrektor) [reply shows/733 - Potluck egres, scraping, Safari in EU.md]: @wesbos thank you!

**Review by randyrektor — COMMENTED** [review]
@wesbos ready to roll

**Comment by wesbos** [comment]
thank you!

**Pull request merged**